src/main/scala/com/microsoft/ml/spark/cognitive/CognitiveServiceBase.scala

```diff
@@ -219,7 +218,7 @@ trait HasInternalJsonOutputParser {
 }
 
 trait HasSetLocation extends Wrappable {
-  override def additionalPythonMethods(): String = {
+  override def pyAdditionalMethods(): String = {
```

Comment:
in a future PR this would be a good idea, this first PR is just moving to inheritance based with minimal semantic changes, then will re-org the structure a bit more